bundles/binding/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/TACmiBinding.java

```diff
@@ -205,8 +208,13 @@ protected void execute() {
                 }
             }
 
+        } catch (SocketTimeoutException e) {
+            logger.info("Receive Timeout on CoE socket");
         } catch (Exception e) {
+            StringWriter w = new StringWriter();
+            e.printStackTrace(new PrintWriter(w));
             logger.warn("Error in execute: {}, Message: {}", e.getClass().getName(), e.getMessage());
+            logger.warn(w.toString());
```

Comment:
Changed logging. As message and class are part of the stacktrace nevertheless, I just log the stacktrace. I also changed the severity to "error", because if an exception arrives here it is very likely an error and not just a warning.